src/ocean/time/StopWatch.d

```diff
@@ -17,6 +17,9 @@
 
```

Comment:
Yeah I guessed so. It's not clear from the commit message, though. Could you add a little more explanation?